packages/console/src/pages/SignInExperience/components/Preview.tsx

```diff
@@ -56,14 +58,18 @@ const Preview = ({ signInExperience, className }: Props) => {
   }, [modeOptions, mode]);
 
   const availableLanguageOptions = useMemo(() => {
-    if (signInExperience && !signInExperience.languageInfo.autoDetect) {
-      return builtInLanguageOptions.filter(
-        ({ value }) => value === signInExperience.languageInfo.fallbackLanguage
-      );
-    }
-
-    return builtInLanguageOptions;
-  }, [signInExperience]);
+    const availableLanguageTags =
+      signInExperience && !signInExperience.languageInfo.autoDetect
+        ? existedLanguageTags.filter(
```

Comment:
Array.find?

Reply:
We need an array here, so derive from the old logic.